x-pack/docs/en/rest-api/ml/jobresource.asciidoc

```diff
@@ -408,6 +413,69 @@ the categorization field value came from.
 For more information, see
 {xpack-ref}/ml-configuring-categories.html[Categorizing Log Messages].
 
+[float]
+[[ml-detector-custom-rule]]
+==== Detector Custom Rule
+
+Custom rules enable you to customize the way detectors operate.
+By configuring rules you can provide domain specific knowledge to the model
+so that the quality of the results better matches a specific use case.
+
+Detectors produce statistically valid anomalies but they have no awareness
+of the domain that is being modeled. For example, a `median(cpu)` detector
+might pick anomalies where the cpu utilization is very small but statistically
+anomalous given the history of a machine. A rule can be employed to instruct
+the detector to skip results where the cpu value is small enough.
+
+Another example would be a `high_count by ip` detector that may result in
+anomalies for IP addresses that are known to be safe. Similarly, a rule can
+instruct the detector to skip results when the IP is in a safe list. Such safe
+lists can be created using `filters` and they can be shared among rules for different jobs.
+
+WARNING: Careless use of rules may result in degrading the quality of the results.
+
+A rule has the following properties:
+
+`actions`::
+  (array) The set of actions to be triggered when the rule applies.
+  The available actions include: +
+  `skip_result`::: the result will not be created. Note this also means this result
+  will not impact the scoring of results. The model be normally updated with the
+  corresponding series value. This is the default action and it serves to skip
+  results that undesired.
```

Comment:
`Note this also means this result will not impact the scoring of results.` - what does this mean?
`The model be normally` -> `The model *will* be normally` or `updated as normal`
`results that undesired` -> `results that are undesired`